testing/sdk_cmd.py

```diff
@@ -32,3 +33,12 @@ def run_cli(cmd, print_output=True):
         log.error(err)
         raise dcos.errors.DCOSException(err)
     return stdout
+
+
+def convert_output_to_list(output):
```

Comment:
The only place this is currently used is where the list of pods of a particular type is returned. Would it not make more sense to wrap `sdk_cmd.run_cli()` with this instead of requiring that this be done at each point where the function is called?